src/ipinfoWrapper.ts

```diff
@@ -351,11 +351,12 @@ export default class IPinfoWrapper {
     }
 
     /**
-     * Get the result of a list of URLs in bulk.
+     * Allow a batch endpoint to group together IPs and retrieve details in bulk.
```

Comment:
The old comment was actually correct. It's a generic endpoint for handling any set of URLs not just IPs.